src/v3/src/transformer/password/transformEnrollPasswordAuthenticator.ts

```diff
@@ -22,43 +22,28 @@ import { PASSWORD_REQUIREMENT_VALIDATION_DELAY_MS } from '../../constants';
 import { getUserInfo } from '../../util';
 import { getUIElementWithName } from '../utils';
 
-const getPasswordMatchingKey = (
-  data: Record<string, unknown>,
-): string | undefined => {
-  if ('credentials.passcode' in data) {
-    return 'credentials.passcode';
-  }
-
-  if ('credentials.password' in data) {
-    return 'credentials.password';
-  }
-
-  if ('credentials.newPassword' in data) {
-    return 'credentials.newPassword';
-  }
-
-  // Should never hit this case as it should be one of values defined above
-  return undefined;
-};
-
 export const transformEnrollPasswordAuthenticator: IdxStepTransformer = ({
   transaction,
   formBag,
 }) => {
   const { nextStep: { relatesTo } = {} } = transaction;
   const passwordSettings = (relatesTo?.value?.settings || {}) as PasswordSettings;
 
-  const { uischema, data } = formBag;
+  const { uischema } = formBag;
 
-  const passwordMatchingKey = getPasswordMatchingKey(data);
-  if (!passwordMatchingKey) {
-    return formBag;
-  }
-
-  const passwordElement = getUIElementWithName(
+  // TODO: remove passwordMatchingKey - still used in fieldKey
+  let passwordMatchingKey = 'credentials.passcode';
+  let passwordElement = getUIElementWithName(
     passwordMatchingKey,
     uischema.elements as FieldElement[],
   ) as FieldElement;
+  if (!passwordElement) {
+    passwordMatchingKey = 'credentials.newPassword';
+    passwordElement = getUIElementWithName(
+      passwordMatchingKey,
+      uischema.elements as FieldElement[],
+    ) as FieldElement;
+  }
```

Comment:
What is this doing? I am not sure if credentials.newPassword is real in the Generic Remediator implementation, so this would break functionality I believe.

Reply:
`credentials.newPassword` is not a thing based on OIE protocol, it will not break anything